document/core/exec/numerics.rst

```diff
@@ -809,11 +833,11 @@ The integer result of predicates -- i.e., :ref:`tests <syntax-testop>` and :ref:
 
 * Let :math:`i` be the result of adding :math:`i_1` and :math:`i_2`.
 
-* Return :math:`\satu_N(i)`.
+* Return :math:`\satu_N(\signed_N(i))`.
```

Comment:
```suggestion
* Return :math:`\satu_N(i)`.
```
This isn't correct here, since the result of i1+i2 is a plain number, not a two's complement representation.